course/assignments/R-intro-lectures-assignments.md

```diff
@@ -0,0 +1,60 @@
+# Week 1 (4 marks) Intro to programming
```

Comment:
Yes, lets make a survey. I make task 4 filling out the survery and move the current 4 into the survey